lib/Caxy/HtmlDiff/Table/TableDiff.php

```diff
@@ -80,11 +82,32 @@ public static function create($oldText, $newText, HtmlDiffConfig $config = null)
      * @param array|null $specialCaseTags
      * @param bool|null  $groupDiffs
      */
-    public function __construct($oldText, $newText, $encoding = 'UTF-8', $specialCaseTags = null, $groupDiffs = null)
+    public function __construct(
+        $oldText,
+        $newText,
+        $encoding = 'UTF-8',
+        $specialCaseTags = null,
+        $groupDiffs = null,
+        $defaultPurifierSerializerCache = null
```

Comment:
I would actually prefer that we don't add this as an argument to the constructor - in release `0.1.0` we added the `HtmlDiffConfig` class that will eventually take the place of all of these constructor arguments once we remove the deprecated configuration calls.

So not adding it to the construct will hopefully encourage use of the `HtmlDiffConfig::setPurifierCacheLocation` method instead.

This will obviously cause issues calling `$this->initPurifier` in the `__construct` here since we won't actually have the purifier path.

I was planning on refactoring the way purifier is utilized soon, and actually move it to be on the `HtmlDiffConfig` object... that might be more work than needed for these changes right now.

For now, I think I'm okay with not exposing the configuration in the constructor and just using the default value for purifier if the __construct is used instead of the static `create` function.